quickstarts/gitlab/config.yml

```
@@ -0,0 +1,51 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: gitlab
+
+# Displayed in the UI (required)
+title: Gitlab
+
+# Long-form description of the quickstart (required)
+description: |-
+  ## What is Gitlab?
+
+  GitLab is a popular single application for the entire DevOps lifecycle, fulfilling all the fundamentals for CI/CD in one environment.
+
+  Now, you can monitor your Gitlab with New Relic, making it easier to get observability into your CI/CD pipeline health and performance.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Monitor Gitlab with the OpenTelemetry in New Relic.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+  - Diogo Pacheco
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - gitlab
+  - opentelemetry
+  - otel
+  - pipelines
+
+dashboards:
+  - gitlab
+
+# Documentation references
+documentation:
+  - name: Set up Gitlab using open telemetry 
+    url: https://github.com/newrelic-experimental/gitlab
+    description: Monitor your Gitlab pipelines with New Relic, making it easier to get observability into your CI/CD pipeline health and performance.
+
+# Content / Design
+icon: logo.svg
+website: https://www.newrelic.com
+
+
```

We'll need to add the `ID` of the install plan to the quickstart config so the quickstart can refer to it!

```suggestion  
installPlans:
    - third-party-gitlab-integration
```